client/src/Components/ChatPage/CardChat.jsx

```diff
@@ -5,30 +5,36 @@ import { Link } from "react-router-dom";
 import { BsCursor } from "react-icons/bs";
 import Form from "react-bootstrap/Form";
 import Button from "react-bootstrap/Button";
-import SendButton from "./SendButton.jsx";
+//import SendButton from "./SendButton.jsx";
```

Comment:
If these things are not used, can we remove these lines instead of commenting them out?

Reply:
done as suggested